packages/solid-crs-manage/lib/features/loan/components/loan-detail.component.ts

```diff
@@ -22,7 +51,35 @@ export class LoanDetailComponent extends RxLitElement {
   render(): TemplateResult {
 
     return html`
-      LOAN DETAIL
+      <nde-large-card .showImage="${false}" .showHeader="${false}">
+        <div slot="content">
+          <h3>
+            ${ this.translator?.translate('loan.detail.card.title')
+    .replace('{{institution}}', this.loanRequest?.from)}
+          </h3>
+          
+          <div id="collection-container">
+            <div>${unsafeSVG(Collection)}</div>
```

Comment:
![image](https://user-images.githubusercontent.com/32491056/153561007-dd797ca5-12f3-4e75-a518-b1e388f7693d.png)

this svg should be dark blue, not black